sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsRestClient.cs

```diff
@@ -605,15 +616,15 @@ internal Response UpdateComponent(string id, string componentPath, string patchD
             }
         }
 
-        internal async Task<Response> SendTelemetryAsync(string id, string dtId, string telemetry, string dtTimestamp = null, CancellationToken cancellationToken = default)
```

Comment:
wow, good catch.

Reply:
This was a bug not a new swagger update right?

Reply:
The service team just changed the parameter names a bit here, and made the messageId a mandatory field. I wouldn't call it a bug, just a feature change